### ISSUE TITLE
[BUG] Hotkeys for 'z' and 'y' not working correctly when using ISO keyboard layout

### ISSUE THREAD

**Issue by EricBlommel** [JohannesKlauss/react-hotkeys-hook#1040]
**Describe the bug**
It doesn't matter if I press 'y' or 'z' on my keyboard. The callbacks for both keys are triggered.

**To Reproduce**

`useHotkeys(['ctrl+z', 'z'], (keys) => console.log(keys));
    useHotkeys(['ctrl+y', 'y'], (keys) => console.log(keys));`

**Expected behavior**
if I press 'y' than only y the callback of y should be triggered. The same for z

**Desktop (please complete the following information):**
 - OS: MacOS Ventura 13.4
 - Browser chrome
 - Version 114.0.5735.133

**Comment by JohannesKlauss** [comment]
Yes, this is related to this [issue](https://github.com/JohannesKlauss/react-hotkeys-hook/discussions/1014). I am working on a version 5 that only listens to the produced key. But I am super busy right now, so this might take some time.

**Comment by JohannesKlauss** [comment]
This would be a breaking change, so it's not feasible to fix it in version 4, sorry.

**Comment by JohannesKlauss** [comment]
This is a layout specific problem that gets addressed in version 5.

This has been fixed in pre release [5.0.0-1](https://www.npmjs.com/package/react-hotkeys-hook/v/5.0.0-1). Please try out this version.
To listen to `y` use you just use `useHotkeys('y', callback, {useKey: true})`

Please let me know if that fixes your issue.
